src/main/java/org/assertj/core/util/Maps.java

```diff
@@ -62,6 +63,15 @@ public static String format(Representation p, Map<?, ?> map) {
       map.put(key, value);
       return map;
   }
-  
+
+  @SafeVarargs
+  public static <K, V> Map<K, V> newHashMapOfEntries(MapEntry<K, V> ... entries) {
```

Comment:
add a unit test for that method in a class named `Maps_newHashMapOfEntries_Test`

Reply:
@joel-costigliola Removing this as similar method was already present: `org.assertj.core.test.Maps#mapOf`

Reply:
good catch!